### PR TITLE
Fix installable build comments on custom statuses

### PR DESCRIPTION
**Pull request by loremattei**
With the recent changes related to the Jetpack app, Peril stopped to update the `Installable Build` comment on PRs. The root cause of the issue is that the job name used for the WordPress and Jetpack installable builds now contains the app name and, so, the status reported by CircleCI doesn't match the expected one anymore. 
Peril expects a `success` status with `ci/circleci: Installable Build` as the context, but the new context is either `"ci/circleci: WordPress Installable Build"` or `"ci/circleci: Jetpack Installable Build"`.

This PR fixes the issue by using regex for the expected statuses.

### CODE REVIEW

File: org/pr/installable-build.ts
Comment (AliSoftware) [inline org/pr/installable-build.ts:137]: Seems like leftover from debugging session there 🙃

(TIL about the `debugger` command, seems useful!)

**Review by AliSoftware — APPROVED** [review]
LGTM… as long as you remove the leftover debugging lines before merging 😛

File: org/pr/installable-build.ts
Comment (loremattei) [reply org/pr/installable-build.ts]: Good catch! 🤦‍♂️ Clearly trying to do too many things in parallel.

**Pull request merged**
